packages/mask/src/extension/popups/pages/Swap/index.tsx

```diff
@@ -31,7 +31,7 @@ const useStyles = makeStyles()((theme) => {
             backgroundColor: theme.palette.background.paper,
             marginLeft: 'auto',
             marginRight: 'auto',
-            height: 720,
+            padding: '25px 0',
```

Comment:
When expand trader list, box's height might be greater than 720.

Reply:
No, you can change `height` to `maxHeight`. if Trader is too much it will exceed the screen height.

Reply:
Add a `maxHeight: 95vh`